roles/composer/tasks/main.yml

```diff
@@ -9,3 +9,9 @@
   command: composer self-update
   register: composer_update_result
   changed_when: composer_update_result.rc == 0 and 'already using composer version' not in composer_update_result.stderr
+
+- name: add Composer vendor binary path
+  lineinfile:
+    dest: /etc/environment
+    regexp: "^PATH=.*"
+    line: PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:./vendor/bin"
```

Comment:
Sadly, that wouldn't work, this file is not evaluated, so variables are not expanded. I'll refine the reflex so it just add at the end of the string instead of rewriting whole.